.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,13 +5,11 @@
 
 ## Related Issue
 <!--- Ideally, new features and changes are discussed in an issue first. -->
-<!--- If there is a corresponding issue, please link to it here: -->
+<!--- If there is a corresponding issue, link to it here. Otherwise, remove this section. -->
```

Comment:
It would be nice if the pull request only showed the new changes. Can you rebase your changes please?

Reply:
Hmm.. I figure I messed up the branch. I haven't done this before. How can I rebase like that? Can you guide me?

Reply:
I think the solution should be the following. origin should be the main vulture repo. 

```
git fetch origin
git checkout test_trial
git rebase origin/master
git push --force
```

(from https://stackoverflow.com/a/45286781/1277298 and https://github.community/t/github-pull-requests-showing-invalid-diff-for-already-merged-branch/559/3)